### ISSUE TITLE
Review terms for H2O2 and hydrogen peroxide

### ISSUE THREAD

**Issue by CuzickA** [PHI-base/phi-eco#42]
There seems to be a term duplication for + H2O2 and + hydrogen peroxide

https://github.com/PHI-base/curation/issues/108

![image](https://github.com/PHI-base/phi-eco/assets/11753634/5e82853b-b79d-4c9c-90fd-9754f4edbe2f)

PECO_0000078 and PECO_0005092

To follow up with @jseager7 in the New Year. Do we need to merge these terms?

**Comment by CuzickA** [comment]
+ hydrogen peroxide PECO_0005092 (created by me) should probably merge into + H2O2 PECO_0000078 (created by Antonia)

**Comment by jseager7** [comment]
@CuzickA I agree that merging is the best solution. You could merge H2O2 into hydrogen peroxide if you think that hydrogen peroxide will be a clearer primary term name, but it doesn't matter all that much if we have exact synonyms.
